Composer/packages/server/src/models/storage/localDiskStorage.ts

```diff
@@ -162,22 +162,39 @@ export class LocalDiskStorage implements IFileStorage {
 
   async zip(source: string, exclusions, cb): Promise<void> {
     const defaultDirectories = [
+      '/Controllers/',
       '/dialogs/',
-      '/language-understanding/',
-      '/language-generation/',
-      '/settings/',
-      '/generated/',
+      '/form-dialogs/',
       '/knowledge-base/',
+      '/language-generation/',
+      '/language-understanding/',
+      '/media/',
+      '/Properties/',
       '/recognizers/',
-      '/form-dialogs/',
+      '/schemas/',
       '/scripts/',
```

Comment:
on my mac it's ```Scripts```, are these case-sensitive? (asking since you have ```Controllers```)

Reply:
The glob pattern we used is case-insensitive. I noticed some of the template bots is `Scripts` and some is `scripts`. maybe need align them in bot templates side;